core/src/services/split-view.js

```diff
@@ -223,6 +223,11 @@ class SplitViewSvcClass {
           IframeHelpers.getIframeContainer().style.paddingBottom = '';
           SplitViewSvc.sendMessageToClients('close.ok');
         });
+    } else {
+      if (comp.get().mfSplitView && comp.get().mfSplitView.settings && comp.get().mfSplitView.settings.onlyExpandBoxVisible) {
+        comp.get().mfSplitView.displayed = false;
+        comp.set({ mfSplitView: comp.get().mfSplitView });
```

Comment:
what about declaring the variable above the if statement: let splitViewState = comp.get().mfSplitView; ?

Reply:
The variable `displayed` is already declared. It comes from  `App.html`. If we are in the if-statement we only should there set the displayed state to false otherwise we'll get side effects and a wrong state.